src/overlays/actors/ovl_En_Ending_Hero/z_en_ending_hero.c

```diff
@@ -39,28 +38,28 @@ void EnEndingHero_Init(Actor* thisx, PlayState* play) {
     this->actor.targetMode = 6;
     this->actor.gravity = -3.0f;
     SkelAnime_InitFlex(play, &this->skelAnime, &object_dt_Skel_00B0CC, &object_dt_Anim_000BE0, this->jointTable,
-                       this->morphTable, 15);
+                       this->morphTable, OBJECT_DT_LIMB_MAX);
     ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 25.0f);
-    func_80C1E748(this);
+    EnEndingHero1_SetupIdle(this);
 }
 
 void EnEndingHero_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void func_80C1E748(EnEndingHero* this) {
-    this->unk244 = 1;
-    this->actionFunc = func_80C1E764;
+void EnEndingHero1_SetupIdle(EnEndingHero* this) {
+    this->isIdle = true;
+    this->actionFunc = EnEndingHero1_Idle;
 }
 
-void func_80C1E764(EnEndingHero* this, PlayState* play) {
+void EnEndingHero1_Idle(EnEndingHero* this, PlayState* play) {
     SkelAnime_Update(&this->skelAnime);
```

Comment:
Went with `Idle` instead of `UpdateSkelAnime` for all EndingHeroX to mimic EndingHero6 and because this is the only action function in these files and it just seemed simpler in this case

Reply:
That makes sense to me.